backend/app/controllers/backoffice/projects_controller.rb

```diff
@@ -16,5 +24,99 @@ def index
         end
       end
     end
+
+    def edit
+    end
+
+    def update
+      if @project.update(update_params)
+        redirect_back(
+          fallback_location: edit_backoffice_project_path(@project.friendly_id),
```

Comment:
Hmm, I see you are using friendly_id for back-office links. I thought differently and was using always UUID at least in back-office as this will be unchangeable, and we don't have to worry about SEO. We should decide to just use one way.

Reply:
No problem, can switch to uuid, I have seen that API is using friendly_id so used same approach, but no big deal to change it here